src/scenes/Header/ProfileToolbar/ProfileToolbar.tsx

```diff
@@ -0,0 +1,59 @@
+import { makeStyles, Typography } from '@material-ui/core';
+import { AccountCircle, MoreVert, NotificationsNone } from '@material-ui/icons';
+import clsx from 'clsx';
+import { FC, useState } from 'react';
+import * as React from 'react';
+import { useSession } from '../../../components/Session/Session';
+import { ProfileMenu } from '../ProfileMenu/ProfileMenu';
+
+const useStyles = makeStyles(({ palette, typography }) => ({
+  name: {
+    fontWeight: typography.fontWeightMedium,
+  },
+  menuWrap: {
+    position: 'relative',
+  },
+  infoWrap: {
+    display: 'flex',
+    alignItems: 'center',
+    justifyContent: 'flex-end',
+    backgroundColor: 'white',
+    boxShadow: '0px 9px 12px rgba(209, 218, 223, 0.43)',
+    width: '250px',
+    padding: '16px',
+    borderRadius: '6px',
+  },
+  icon: {
+    marginLeft: '12px',
+    cursor: 'pointer',
+    '&:last-child': {
+      marginLeft: '6px',
+    },
+  },
+  iconProfile: {
+    fill: palette.secondary.main,
+  },
+}));
+
+export const ProfileToolbar: FC = () => {
+  const classes = useStyles();
+  const [session] = useSession();
+  const [isProfileMenuOpen, setProfileMenuOpen] = useState(false);
+
+  return (
+    <div className={classes.menuWrap}>
```

Comment:
I would just use a fragment here instead.
```suggestion
    <>
```